Lib/argparse.py

```diff
@@ -89,6 +89,7 @@
 import sys as _sys
 
 from gettext import gettext as _, ngettext
+from distutils.util import strtobool as _strtobool
```

Comment:
Let's not take a dependency on distutils here - copy the function in (and add the forced `bool` conversion if necessary).

Reply:
Done.
And instead of forcing the cast to bool, the function returns True/False of bool type.